packages/web/pages/assets/index.tsx

```diff
@@ -33,6 +33,7 @@ import {
 } from "../../hooks";
 import { WalletConnectQRModal } from "../../modals";
 import { EventName } from "../../config";
+import { useTokenTransferQueryParams } from "../../hooks/window/use-assets-transfer-query-params";
```

Comment:
Add `use-assets-transfer-query-params` to the index file in the hooks/window folder. Then just import it in the existing `../../hooks` import above.

Reply:
clean